db/dcrsqlite/apisource.go

```diff
@@ -1307,11 +1307,21 @@ func (db *wiredDB) GetExplorerTx(txid string) *explorer.TxInfo {
 	return tx
 }
 
-func (db *wiredDB) GetExplorerAddress(address string, count, offset int64) *explorer.AddressInfo {
+func (db *wiredDB) GetExplorerAddress(address string, count, offset int64) (*explorer.AddressInfo, error) {
 	addr, err := dcrutil.DecodeAddress(address)
+
+	// checks for P2Pk type addresses
+	if strings.HasPrefix(address, "Dk") || strings.HasPrefix(address, "Tk") || strings.HasPrefix(address, "Sk") {
+		return nil, fmt.Errorf("P2PK address detected")
+	}
 	if err != nil {
 		log.Infof("Invalid address %s: %v", address, err)
-		return nil
+
+		// This is here to detect a bitcoin type address
+		if (strings.HasPrefix(address, "bc") || strings.HasPrefix(address, "1") || strings.HasPrefix(address, "3")) && len(address) >= 25 && len(address) <= 34 {
+			return nil, fmt.Errorf("Bitcoin address detected")
```

Comment:
Possible Bitcoin address detected.

We do no validation of the suspected bitcoin address, so we should soften our language.